serve.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from paste.deploy import loadapp
+from waitress import serve
+
+if __name__ == "__main__":
+    port = int(os.environ.get("PORT", 5000))
+    app = loadapp('config:release.ini', relative_to='.')
+    serve(app, host='0.0.0.0', port=port)
```

Comment:
should the host be hard-coded? don't worry about this if it's not a quick fix

Reply:
this one i follow instruction from the pyramid cookbook, so I don't really know how to do otherwise for now.